pilot/docker/Dockerfile.proxyv2

```
@@ -11,6 +11,9 @@ FROM docker.io/istio/base:${BASE_VERSION} as default
 # obtained from debian ca-certs deb using fetch_cacerts.sh
 ADD ca-certificates.tgz /
 
+# Add WASM plugins for stats and metadata_exchange
+ADD wasm /
```

`hadolint` is failing with this error:
```
$ hadolint Dockerfile.hadolint 
Dockerfile.hadolint:15 DL3020 Use COPY instead of ADD for files and folders
```

```suggestion
COPY wasm /
```

---

@gargnupur ^^^